powerhome/playbook
# [PLAY-87] Upgraded FontAwesome to 6.1.2

![React Kit Coverage](https://img.shields.io/badge/React%20Kit%20Coverage-14.04%25-red)#### Screens
![Screen Shot 2022-07-29 at 4 21 02 PM](https://user-images.githubusercontent.com/9158723/181837563-4f17704c-4545-4388-9035-dc95f0079cac.png)


#### Breaking Changes

No. This should be tested by Ninjas. But everything from 6 is [Backwards Compatible](https://fontawesome.com/v6/docs/web/setup/upgrade/#upgrade-from-version-5-when-using-a-kit), including retaining old icon names correctly.

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/PLAY-87

#### How to test this
Test with the icon kit inside of playbook and try out some old icons from [this list](https://fontawesome.com/v6/docs/web/setup/upgrade/whats-changed#icons-renamed-in-version-6) to make sure they still show up correctly. 

#### Checklist:

~- [ ] **SPECS** Please cover your changes with specs.~
